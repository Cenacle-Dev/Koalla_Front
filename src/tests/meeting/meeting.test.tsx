import { render } from "@testing-library/react";
import Meeting from "../../components/meeting/meeting";

describe("모임 카드 테스트", () => {
	let title: any;
	let body: any;
	let tag: any;
	it("모임 카드 스냅샷", () => {
		title = "모임1";
		body = "모임 내용입니다.";
		tag = "#태그1";
		const view = render(<Meeting title={title} body={body} tag={tag} />);
		expect(view.container).toMatchSnapshot();
	});
});
