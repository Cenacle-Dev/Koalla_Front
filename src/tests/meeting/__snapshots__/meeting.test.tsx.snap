// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`모임 카드 테스트 모임 카드 스냅샷 1`] = `
<div>
  <li
    class="sc-bdvvtL ewUwcx"
  >
    <img
      alt=""
      class="sc-gsDKAQ hjZJNh"
      src="https://media.istockphoto.com/photos/mug-on-plate-filled-with-coffee-surrounded-by-coffee-beans-picture-id157528129?s=612x612"
    />
    <div
      class="sc-dkPtRN sgIFf"
    >
      <h3
        class="sc-hKwDye gMqeAw"
      >
        모임1
      </h3>
      <p
        class="sc-eCImPb glBdsk"
      >
        모임 내용입니다.
      </p>
      <p
        class="sc-jRQBWg kgwKyj"
      >
        #
        #태그1
      </p>
    </div>
  </li>
</div>
`;
